viite-UI/src/view/MapView.js

```diff
@@ -130,7 +131,18 @@
 
     //when the map dragging stops the cursor value returns to the initial one
     map.on('pointerup', function(evt) {
-      map.getViewport().style.cursor = "initial";
+      map.getViewport().style.cursor = oldCursorStyle;
+    });
+
+    eventbus.on('keydown', function(tool){
```

Comment:
think this causes cursor constantly switch between  two cursor styles when adding links to selection.